cfr_tool/explosives.py

```diff
@@ -0,0 +1,136 @@
+import re
+import clean_text as ct
+
+class Explosives:
+    def __init__(self, db, soup):
+        self.db = db
+        self.soup = soup
+
+    def create_load_explosives(self):
```

Comment:
This is great. I might even go a step further and turn this into a common interface. 

So, devise a set of common functions that each module will use (e.g. init/create, load outer packaging codes, load inner packaging codes). Then, over time, you build out classes that are formatted the same way that parse through different sections of the CFR. Maybe ABS could even work on this too and plug in parsing code of their own.

I specifically commented here because `create_load_explosives` is the least general-sounding function name I see in here.